### ISSUE TITLE
While writing text, pressing Enter on #tag selection removes camel casing

### ISSUE THREAD

**Issue by xarvh** [elk-zone/elk#3156]
I'm trying to use CamelCase whenever I write tags, because it's the accessible thing to do.
However, if I'm not careful, Elk's tag autocomplete/selector will remove camel casing from the tag name I just entered.

Expected behavior: when selecting a tag the camel casing that the user entered manually should be preserved.

**Comment by audunmb** [comment]
This is a duplicate of #2414
